src/box/user.cc

```diff
@@ -670,6 +670,16 @@ user_find_by_name(const char *name, uint32_t len)
 	return NULL;
 }
 
+/** Return user name by id */
+const char *
+user_name_by_id(uint32_t uid, uint32_t *name_len)
+{
+	struct user *user = user_find(uid);
+	const char *name = user->def->name;
```

Comment:
It seems, we get a NULL dereference here if the user doesn't exist.

Reply:
Check for `NULL` added.